### ISSUE TITLE
preventDefault on a tags

### ISSUE THREAD

**Issue by karsasmus** [claviska/jquery-selectable#1]
preventDefault on a tags doesn't work.

(It breaks post preview on Leafpub)

**Comment by claviska** [comment]
Do you have an example? Is there anything in the error console?

**Comment by karsasmus** [comment]
If I click on a post to preview in the right tab, the post gets loaded in edit mode.

**Comment by claviska** [comment]
If I'm understanding you correctly, I don't think it's related to this plugin. I'm using it for a couple different things (Postleaf beta7 included) and it's working fine.

Have any changes been made to the script for that section? Check the error console to make sure nothing's triggering an error. It could be related to the name change.

**Comment by karsasmus** [comment]
I tested all files leafpub files (scripts/modules) against postleaf files (I built everytimte via gulp build).
I changed your selectable@1.0.4 to 1.0.3 and everything worked.

I made no changes except Postleaf --> Leafpub

**Comment by claviska** [comment]
Ah, shoot, looks like it's due to [this change](https://github.com/claviska/jquery-selectable/commit/953a5b854313af5c81fc388862a0b8b24f2d516b#diff-c3ca8254274c177f2822fdf0af5994c8L72). Sorry, forgot that was even pushed last week.

Could you change line 72-75 to this and see if that resolves it?

```js
// Prevent clicks on links from hijacking the page
if( $(event.target).parents().addBack().is('a') ) {
    event.preventDefault();
}
```

If so, I'll push a quick bug fix.

**Comment by karsasmus** [comment]
That worked, thank you :slightly_smiling_face:

**Comment by claviska** [comment]
Fixed in [1.0.5](https://github.com/claviska/jquery-selectable/releases/tag/1.0.5).
